app/lib/irs8812_ty2021_pdf.rb

```diff
@@ -6,10 +6,7 @@ def source_pdf_name
   end
 
   def initialize(submission)
-    @submission = submission
-    @qualifying_dependents = submission.qualifying_dependents
-    @ctc_qualifying_dependents =  @qualifying_dependents.select(&:qualifying_ctc?)
-    @benefits = Efile::BenefitsEligibility.new(tax_return: submission.tax_return, dependents: @qualifying_dependents)
+    @xml_document = SubmissionBuilder::Ty2021::Documents::Schedule8812::Base.new(submission).document
```

Comment:
Like... could we potentially run into that here? Do any of these values in base that are consumed diverge in the PR case?

Reply:
This always looks at Base, but seems like there is both a Base and PR version of the 8812. Do we need that?

When there is divergence in those docs, maybe we should create a method so that we can get the right method here...